### DISCUSSION TITLE
exec ${nextRelease.version} is always 1.0.0?

### DISCUSSION THREAD

**Discussion by meza** [semantic-release/semantic-release#2494] [Q&A] [answered]
Hey,

I'm using semantic-release to release a non-js piece of software and I need to do some version rewriting in arbitrary files.
I've got this in my .releaserc.json
```json
[
...
      "@semantic-release/exec",
      {
        "prepareCmd": "./scripts/release.sh \"${nextRelease.version}\""
      }
]
```

All the other steps work just fine, even the tagging does too (which also uses the nextrelease.version) 
![image](https://user-images.githubusercontent.com/211977/176960474-a101d3a4-9ada-427a-89b3-55108c055a03.png)

But for whatever reason, the result of the prepareCmd is always a version 1.0.0.

This is my release script:

```sh
#!/bin/sh

VERSION=$1

echo "Replacing version with ${VERSION}"
sed -e "s/VERSION/${VERSION}/" -i modinfo.json
```

The result: Version: 1.0.0 in the json.. :(

Any ideas why?

**Comment by meza** [comment, marked as answer]
User error, mea culpa

**Reply by khalidkhushal** [reply]
I am facing the same issue.
What kind of user error is this, can you please provide with a little more details?

**Reply by meza** [reply]
I wish I could remember, it was a long time ago. I might have simply had the 1.0.0 hardcoded in the json instead of the replace token that sed was looking for.
